intellij-toml/core/src/main/kotlin/org/toml/ide/annotator/TomlAnnotator.kt

```diff
@@ -24,5 +24,10 @@ class TomlAnnotator : AnnotatorBase() {
                 holder.newAnnotation(HighlightSeverity.ERROR, "Inline tables are intended to appear on a single line").create()
             }
         }
+
+        val parent = element.parent
+        if (element.text == "," && parent is TomlInlineTable && element.textOffset > parent.entries.last().textOffset) {
```

Comment:
Let's better use `lastOrNull` here not to produce exception if there are no entries.
Yeah, looks like it's impossible for now but anyway